examples/tests/allowed/custom_s3_zip_file.yml

```diff
@@ -0,0 +1,11 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Transform: AWS::Serverless-2016-10-31
+Description: Hello SAM
+
+Resources:
+  s3File:
+    Type: Custom::S3ZipFile
+    Properties:
+      Bucket: bucket
+      Key: example.txt
```

Comment:
why is different different from the path in the uri?

Isn't key the file path in s3?